### ISSUE TITLE
The way you use VTP-DS pipeline in this repo?

### ISSUE THREAD

**Issue by npk7264** [KiseKloset/DM-VTON#12]
Hi authors, I am wondering how you use VTP-DS pipeline in this repo? 
Because first I thought you use this to create dataset, then training, but now I think I need your explaination about the way you use VTP-DS. Could you please help me understand VTP-DS and give me the place you use it in this repo?
Thank you!

**Comment by zero-nnkn** [comment]
Hi @npk7264 , 
VTP-DS aims to augment training data from images in the original data and new poses. We believe that it can help VITON model is more robust to data in the wild. However, in VITON dataset, it does not seem to help improve the results on the original test set (which only has simple poses). 
The model we use to generate data: [PIDM](https://github.com/ankanbhunia/PIDM)
